src/platform/site-wide/maintenance-banner/tests/01-accessibility.e2e.spec.js

```diff
@@ -0,0 +1,132 @@
+const E2eHelpers = require('../../../testing/e2e/helpers');
```

Comment:
Ignore this, sorry brought over from boilerplate code and will omit asap.

Reply:
Is this file still necessary?

Reply:
Yupp, it will contain the e2e tests today 🙂